src/components/timeline/ActiveTabResourcesPanel.js

```diff
@@ -63,6 +68,11 @@ class ActiveTabResourcesPanel extends React.PureComponent<Props> {
           })}
         >
           Resources ({resourceTracks.length})
+          <TrackThread
```

Comment:
As a potential follow-up: We should consider what the height of the ThreadActivityGraph means here for a merged thread. I'm assuming it's just doing the naive thing, which means the height of the graph will be much smaller than it should be.

Maybe we can chat about this.